projects/cypress/CY8CKIT_064S0S2_4343W/mtb/aws_tests/afr.mk

```diff
@@ -356,7 +356,18 @@ INCLUDES+=\
 	$(CY_AFR_ROOT)/libraries/c_sdk/aws/defender/src\
 	$(CY_AFR_ROOT)/libraries/c_sdk/aws/defender/src/private\
 	$(CY_AFR_ROOT)/libraries/coreMQTT/source/include\
-	$(CY_AFR_ROOT)/libraries/coreMQTT/source/portable\
+    $(CY_AFR_ROOT)/libraries/coreMQTT/source/portable
```

Comment:
Is this the only board you are updating for the shadow integration tests?

Reply:
sure

Reply:
I will delete the final two because they exist above already.